classes/Tools.php

```diff
@@ -2509,9 +2509,10 @@ public static function generateHtaccess($path = null, $rewrite_settings = null,
         fwrite($write_fd, "AddType application/vnd.ms-fontobject .eot\n");
         fwrite($write_fd, "AddType font/ttf .ttf\n");
         fwrite($write_fd, "AddType font/otf .otf\n");
+        fwrite($write_fd, "AddType font/woff2 .woff2\n");
```

Comment:
Then there is a mistake to fix in the [Tools.php](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Tools.php#L2361) of the develop branch.